rpc/jsonrpc_state_server.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package rpc
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/ava-labs/avalanchego/trace"
+)
+
+type stateReader interface {
+	Tracer() trace.Tracer
+	ReadState(ctx context.Context, keys [][]byte) ([][]byte, []error)
```

Comment:
nit: Can we call this `GetStateKey`?

Reply:
This is the previously used name, so I think it's better to follow the current style for this PR